build.gradle

```diff
@@ -19,10 +19,10 @@ ext {
   if (variant == 2.4) {
     groovyVersion = "2.4.15"
     minGroovyVersion = "2.4.0"
-    groovyDependency = "org.codehaus.groovy:groovy-all:${groovyVersion}"
+    groovyDependency = ["org.codehaus.groovy:groovy-all:${groovyVersion}"]
   } else if (variant == 2.5) {
     groovyVersion = "2.5.2"
-    minGroovyVersion = "2.5.0"
+    minGroovyVersion = "2.5.2"
```

Comment:
@szpak I think I've upgraded the dependency because of this issue https://issues.apache.org/jira/browse/GROOVY-8710. You can certainly try to downgrade it and see if it still works.